### PR TITLE
Change url links to latest url app

### PR DESCRIPTION
**Pull request by amanalili**
Task: edge-login-UI-web: wrong url to app

### CODE REVIEW

File: packages/edge-login-ui-react/src/locales/default.js [hunk 1]
```diff
@@ -852,24 +852,24 @@ const strings = {
...
   playstore_link: 'https://play.google.com/store/apps/details?id=com.airbitz',
-  logo_white_link: 'https://airbitz.co/static/img/logo-nav.png',
+  logo_white_link: 'https://edge.app/static/img/logo-nav.png',
   designed_by: 'Designed and Built by Edge in',
   company_location: 'San Diego, California, USA',
 
   request_footer:
     'Sent from the Edge Mobile Bitcoin Wallet + Business Directory',
-  request_footer_link: 'https://airbitz.co/app',
+  request_footer_link: 'https://edge.app/app',
```
Comment (paullinator) [inline packages/edge-login-ui-react/src/locales/default.js]: Change this to just https://edge.app

File: packages/edge-login-ui-react/src/locales/default.js [hunk 2]
```diff
@@ -852,24 +852,24 @@ const strings = {
...
   request_footer_link_title: 'Download Edge for iOS or Android',
   request_footer_contact:
-    'mailto:info@airbitz.co?subject=Edge+Bitcoin+Email+Request',
+    'mailto:info@edge.app?subject=Edge+Bitcoin+Email+Request',
```
Comment (paullinator) [inline packages/edge-login-ui-react/src/locales/default.js]: Make subject =Edge+Support+Request

File: packages/edge-login-ui-react/src/locales/default.js [hunk 3]
```diff
@@ -852,24 +852,24 @@ const strings = {
 
   app_support_requested: 'Support Requested',
   app_name: 'Edge',
-  app_url_prefix: 'airbitz',
-  app_homepage: 'https://airbitz.co',
-  app_support_email: 'support@airbitz.co',
+  app_url_prefix: 'edge',
```
Comment (paullinator) [inline packages/edge-login-ui-react/src/locales/default.js]: Where is app_url_prefix used? Did you test that this still works?

Comment (amanalili) [reply packages/edge-login-ui-react/src/locales/default.js]: Searching that yields no results on the code, must be from the old codes. Yup everything works fine

**Pull request merged**